### ISSUE TITLE
wrong details

### ISSUE THREAD

**Issue by ameet20** [pytorch/pytorch#17218]
Hello,

The explanation of `pos_weight` for [BCEWithLogitsLoss](https://pytorch.org/docs/stable/nn.html#torch.nn.BCEWithLogitsLoss) mentions:
`where :math:`p_n` is the positive weight of class :math:`n`.`

We don't have `n` classes (this is a binary loss). The correct description would be something like this:
`:math:p_n is the weight of the positive class for sample n in the batch`

(I also don't understand the utility of having a different positive-class-weight per sample in a batch, because you usually apply the same class weight for the whole training data. But that's a different issue.)

**Comment by velikodniy** [comment]
There was a typo, I agree. The correct description would be `:math:p_c is the weight of the positive class for class :math:c`. The proposed by @ameet20 text is wrong actually.

The purpose of `pos_weight` is to control precision/recall in _multi-label_ classification where `c` may be > 1.
